stackslib/src/net/inv/epoch2x.rs

```diff
@@ -2301,6 +2302,7 @@ impl PeerNetwork {
                     stats.done
                 );
                 if !stats.done {
+                    // TODO: the result of this match doesn't seem to be used? is that intentional?
```

Comment:
Looks like this result is intentionally ignored. @jcnelson can you confirm?

Reply:
Yes; this was written in a time when I was a lot less idiomatic in my Rust code and when we didn't really prioritize conforming to it.

Reply:
Thanks Jude. So, @jferrant, how about modifying this so the result of that expression is `()` instead of a bool, then it won't look suspicious and you can remove the TODO.